packages/connection/src/Manager.php

```diff
@@ -56,11 +56,16 @@ public function is_active() {
 	 * Returns true if the user with the specified identifier is connected to
 	 * WordPress.com.
 	 *
-	 * @param Integer $user_id the user identifier.
+	 * @param Integer|Boolean $user_id the user identifier.
 	 * @return Boolean is the user connected?
 	 */
-	public function is_user_connected( $user_id ) {
-		return $user_id;
+	public function is_user_connected( $user_id = false ) {
```

Comment:
should we deprecate Jetpack::is_user_connected on this PR?

Reply:
I'm hesitant to do it because the Connection Manager still has several non-implemented methods, and we don't want to fool someone that it is the canonical way of working with the connection. 

Perhaps in a follow-up PR where migrate the legacy usages to use the connection package?

Reply:
👍